### DISCUSSION TITLE
Configuration of central server question

### DISCUSSION THREAD

**Discussion by hacsjalano** [lbbrhzn/ocpp#71] [Q&A] [answered]
Hello!  I'm just getting started - forgive the newbie question, but I can't quite seem to understand where to do the steps in README.md, Installation Step 3: Configure the Central System.

It appears someplace I can change the default port and select which measurands for my sensor entities, but I don't see where this configuration panel (or file) is.

I have looked under the HACS->Integrations->ocpp tile, but the 3 dots under that only show Information, Update information, Reinstall, Open issue, Request for removal and Uninstall.

Under Configuration->Integrations I only see HACS.  Is there some place I'm missing that is where the OCPP central server is configured?

Thanks again for doing this project!

**Comment by hacsjalano** [comment, marked as answer]
Okay, I'm answering my own question.  It appears that after Step 2, Add the OCPP integration, I needed to clear my browser cache.  The panel for the integration was not immediately visible, but it showed up after I did that (I am using Firefox on a Mac).  I discovered this after hitting the configuration page with a different browser which did not have that cached!

**Reply by lbbrhzn** [reply]
The configuration window should pop up when you add the OCPP integration. The labels are translated in the current language of the home assistant user, but we only support English and Dutch at the moment. It should default to English if the language is not supported.
What was your language setting?

**Reply by hacsjalano** [reply]
That's actually probably my fault the config window didn't pop up - I actually downloaded this integration about 1-2 weeks ago, before a lot of the current version changes were in place.  Back then, the configuration instructions were to add the settings in the .yaml file.  Well, I couldn't get that to work probably because the code was still being refactored, so I just let it sit in my system.  Then when I checked yesterday I noticed the README.md instructions were updated with the new configuration, I hit "upgrade" on my homeassistant web interface and got the new version!  So I believe because I did it that way it skipped over the usual install procedure.

In any event, the central server appears to be up and running!  Oh, and my language setting is English.  Thanks for the quick reply!

**Reply by lbbrhzn** [reply]
Hi Hacsjalano, what charger are you using with ocpp? Just to add to the list of supported chargers...

**Reply by hacsjalano** [reply]
Unfortunately my charger hasn't arrived yet.  It's back-ordered.  I setup my homeassistant and OCPP central server all ready for it to connect my future charger, and now I won't be seeing it for several months!!  Blame the world-wide chip shortage.  So I'm researching other options right now.  I'll follow up when I actually get one in my hands!

**Comment by pasolobo2** [comment]
The same thing happens to me, I just installed the plugin but it does not appear in Configuration -> Integrations. My language is Spanish. I have tried to change it to English, but nothing, I can not access to configure. I have also tried older versions of the plugin. What could be happening to me?

**Reply by lbbrhzn** [reply]
after installing OCPP through HACS, you need to restart home assistant.

**Reply by pasolobo2** [reply]
Yes. I rebooted my home assistant several times. I've searched the HomeAssistant logs but can't find any errors. Any ideas?. Thanks
